components/usage/pkg/db/types.go

```diff
@@ -80,9 +79,21 @@ func (n VarcharTime) IsSet() bool {
 
 // Value implements the driver Valuer interface.
 func (n VarcharTime) Value() (driver.Value, error) {
-	return n.t.UTC().Format(time.RFC3339Nano), nil
+	if n.IsSet() {
+		return TimeToISO8601(n.t), nil
+	}
+	return "", nil
 }
 
 func (n VarcharTime) String() string {
```

Comment:
This is what is used to print this field for humans, when we're converting the struct to a string. We align with the DB value and also print ISO8601